lib/babe/build_integration_test.go

```diff
@@ -364,8 +324,6 @@ func TestDecodeExtrinsicBody(t *testing.T) {
 	require.True(t, contains)
 }
 
-// TODO: add test against latest dev runtime
-// See https://github.com/ChainSafe/gossamer/issues/2704
 func TestBuildBlockTimeMonitor(t *testing.T) {
```

Comment:
This is just testing some metrics?

Reply:
Yeah think so. Perhaps this is another one I can investigate in follow up PR if you think its worth it